code/modules/projectiles/ammunition/_firing.dm

```diff
@@ -2,25 +2,26 @@
 	distro += variance
 	var/targloc = get_turf(target)
 	ready_proj(target, user, quiet, zone_override, fired_from)
+	var/obj/projectile/thrown_proj
 	if(pellets == 1)
 		if(distro) //We have to spread a pixel-precision bullet. throw_proj was called before so angles should exist by now...
 			if(randomspread)
 				spread = round((rand() - 0.5) * distro)
 			else //Smart spread
 				spread = round(1 - 0.5) * distro
-		if(!throw_proj(target, targloc, user, params, spread, fired_from))
+		thrown_proj = throw_proj(target, targloc, user, params, spread, fired_from)
+		if(!thrown_proj)
```

Comment:
```suggestion
		if(isnull(thrown_proj))
```

do it like this since you changed `throw_proj()` to return an object instead of a boolean value, much more explicit and all